cli/mender-artifact/partition.go

```diff
@@ -304,6 +352,18 @@ type fatFile struct {
 	tmpf          *os.File
 }
 
+func newFatFile(imageFilePath string, partition partition) (*fatFile, error) {
+	tmpf, err := ioutil.TempFile("", "mendertmp")
```

Comment:
Suggestion: Maybe we could stick to some convention - mendertmp-FILENAME or something to track better the resource leaks if happening?

Reply:
Great idea! simple and easy. Like it.